### PR TITLE
A couple of fixes for Octane benchmarks

### PR DESCRIPTION
**Pull request by baxtersa**

### CODE REVIEW

File: stopify-continuations/src/callcc/callcc.ts [hunk 1]
```diff
@@ -146,7 +146,7 @@ const visitor: Visitor = {
         t.callExpression(t.identifier('require'),
           [t.stringLiteral('stopify/dist/src/stopify/compileFunction')]) :
         // provided by dist/stopify-compiler.bundle.js
-        t.memberExpression(t.identifier('stopifyCompiler'), t.identifier('module'));
```
Comment (arjunguha) [inline stopify-continuations/src/callcc/callcc.ts]: What's this for?

Comment (baxtersa) [reply stopify-continuations/src/callcc/callcc.ts]: The module exports are just properties of `stopifyCompiler`, there is no `module` property. I'm not sure how the `eval` unit tests on master are passing without this change.

Comment (baxtersa) [reply stopify-continuations/src/callcc/callcc.ts]: To follow up, the eval unit tests run under `node` and are compiled with `--require-runtime`, so the global free variable `stopifyCompiler` is replaced with `require('stopify/dist/src/stopify/compileFunction')`.

**Comment by arjunguha** [comment]
Rebased so we can merge easily.

**Pull request merged**